examples/heatmap/adapter/heatmap_client.py

```diff
@@ -1,50 +1,56 @@
-from turtle import st
-from formant.sdk.agent.v1 import Client as AgentClient
 import os
+import datetime
 import time
 from random import randint
 
+from formant.sdk.agent.v1 import Client as AgentClient
+
+
+AGENT_URL = "unix:///var/lib/formant/agent.sock"
+LOCATION_STREAM_NAME = "heatmap.point.location"
+NUMERIC_STREAM_NAME = "heatmap.point.weight"
+PUBLISH_FREQUENCY = 30
 
 class HeatmapClient:
     def __init__(self) -> None:
-        agent_url = "unix:///var/lib/formant/agent.sock"
-        self.agent_url = os.getenv("AGENT_URL", agent_url)
+        self.agent_url = os.getenv("AGENT_URL", AGENT_URL)
```

Comment:
This was necessary for an older version of the agent, we should update the agent-sdk in the requirements, and just remove this line and the ones associated with it

Reply:
Good to know; do you know which version doesn't require this anymore? Or what's a reasonable minimum version to have?

Reply:
I think just updating to the latest should be ok (1.111.67)